src/fs/rock/RockSwapDir.cc

```diff
@@ -345,6 +349,12 @@ Rock::SwapDir::parse(int anIndex, char *aPath)
     validateOptions();
 }
 
+bool
+Rock::SwapDir::smpAware() const
+{
+    return UsingSmp();
```

Comment:
Please undo this change. If this was required to fix test cases, then we need a different solution for fixing test cases.